README.md

```diff
@@ -1,3 +1,5 @@
 # test-01
 
 jhhjkjhjkhkjs
+
+;wjkjewhkjeh
```

Comment:
It looks like you've added a random string in the README. Could you please elaborate on the purpose of this addition?